utils/authenticator-helper.js

```diff
@@ -0,0 +1,22 @@
+const atob = require('atob');
+
+function parseJwt(token) {
+  if (!token || !token.includes('.')) { return null; }
+  try {
+    const base64Url = token.split('.')[1];
+    const base64 = base64Url.replace(/-/g, '+').replace(/_/g, '/');
+    const jsonPayload = decodeURIComponent(atob(base64)
+      .split('')
+      .map(c => `%${`00${c.charCodeAt(0).toString(16)}`.slice(-2)}`)
+      .join(''));
+
+    return JSON.parse(jsonPayload);
+  } catch (error) {
+    console.error('Session token parsing unexpected error:', error);
+    return null;
+  }
+}
+
+module.exports = {
+  parseJwt,
+};
```

Comment:
Don't know where to leave this comment. When I run `npm i` on a fresh generated project, I got this warning: 

```
npm WARN deprecated superagent@3.7.0: Please note that v5.0.1+ of superagent removes User-Agent header by default, therefore you may need to add it yourself (e.g. GitHub blocks requests without a User-Agent header).  This notice will go away with v5.0.2+ once it is released.
```

Reply:
I set the user agent in the api service (Where it is interesting). The warning can't be removed until they release the new version 😞